lib/assets.js

```diff
@@ -21,20 +21,16 @@ function asset(options = { inline: [] }) {
 }
 
 
-const loadAssetsMap = () => {
+
+const loadAssetsMap = (origin) => {
   assetsMap = {};
 
-  const entryPath = resolve('app/assets/entry.html');
-  const distDir = 'public/dist';
-  const entryDir = dirname(entryPath);
-  const entryName = basename(entryPath);
-  const resultPath = resolve(distDir, entryName);
-  const entryPaths = getAssetPaths(fs.readFileSync(entryPath, 'utf8'));
-  const resultPaths = getAssetPaths(fs.readFileSync(resultPath, 'utf8'));
+  const entryDir = 'app/assets';
+  const map = JSON.parse(fs.readFileSync(resolve('assetsMap.json'), 'utf8'));
```

Comment:
What about renaming assetsMap.json to something like: `hashed-assets-map.json` or `esbuild-map.json`?

Also, we could make the map a `.js` file which exports the object so that we don't have to parse it when we read it into memory. Instead we could just require it.